packages/theme/components/Content/ImageGallery.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <SfSection>
+    <div class="grid grid-images"
+    v-if="isLayoutReversed"
+    >
+      <div class="grid__row">
+        <div class="grid__col">
+          <SfImage
+            class="image"
+            :src="section.imgTwoXl"
+            alt=""
+            @click="router.push('/c/'.concat(section.link[1]))"
+          />
+        </div>
+      </div>
+      <div class="grid__row">
+        <div class="grid__col medium">
+          <SfImage
+            class="image"
+            :src="section.imgOneMd"
+            alt=""
+            @click="router.push('/c/'.concat(section.link[3]))"
```

Comment:
Could we wrap the SfImage tag in `<a href="xxx">` or instead of adding a js callback?

Reply:
Looking at this PR, we probably shouldn't have any @click event handlers or uses of `useRouter` at all